src/TemplateCompiler/NodeFactory/VarDocNodeFactory.php

```diff
@@ -10,18 +10,37 @@
 
 final class VarDocNodeFactory
```

Comment:
Not particular proud or feel like the approaches taken here is the correct one. But with my limited understanding of this project, this was the way to go for now.